src/utils.c

```diff
@@ -363,3 +363,34 @@ SEXP coerceUtf8IfNeeded(SEXP x) {
   return(ans);
 }
 
+// #4327 this function converts an AsIs object x to list(x) or list(list(x))
+// so that they can be preserved "as is". In addition, the "AsIs" class
+// will be removed.
+SEXP protectAsIs(SEXP x, bool dblist) {
```

Comment:
`dblist` parses as `DB list` to me, maybe `listlist` or `dbl_list` or `relist`?

Reply:
changed to `twolists`

Reply:
works for me